### DISCUSSION TITLE
Volume control not working on iMac 2010 used as external monitor

### DISCUSSION THREAD

**Discussion by kilianekamp** [MonitorControl/MonitorControl#1619] [General]
### Before opening the issue, have you...?

- [X] Searched for existing issues
- [X] Read through [the Monitor-Troubleshooting Wiki](https://github.com/MonitorControl/MonitorControl/wiki/Monitor-Troubleshooting)
- [X] Updated MonitorControl to the latest version (if applicable)

### Describe the issue

The volume control is not working on my iMac 2010 27" that I use as an external monitor to my MacBook Air M3. The volume is always maxed out. I can move the slide in the MonitorControl app, but there's no actual change to the volume. Brightness control works just fine.

### Expected behavior

I would like to change the actual volume of my monitor by moving the volume control slider.

### Anything else?

These are my settings. If I uncheck "Use hardware DDC control", the volume slider disappears from the app.

![Screenshot 2024-09-24 at 21 33 16](https://github.com/user-attachments/assets/ca8b0041-52cc-4dbe-91ba-28f7f584d8aa)

Thank you for creating MonitorControl!

### Environment Information (please complete the following information)

```markdown
- macOS version: Sonoma 14.6
- Mac model: MacBook Air M3, 15", 2024
- MonitorControl version: 4.2.0 Build 7048
- Monitor(s): iMac Mid 2010, 27"
- Monitor Cable(s)/Connection(s): USB-C to Mini Displayport
- Apple Silicon/M1 (yes or no): Yes (M3)
```

**Comment by waydabber** [comment]
The 27" iMac is not DDC compatible (in Target Display Mode) and does not support DDC volume control. This is why volume control does not work.

**Reply by kilianekamp** [reply]
Interesting, thank you! Why does this not affect the brightness control? Is there any workaround you could think of?
